scripts/scil_score_tractogram.py

```diff
@@ -0,0 +1,376 @@
+    for i, roi in enumerate(comb_filename):
+        mask_1_filename, mask_2_filename = roi
+
+        # That would be done here.
+        # Automatically generate filename for Q/C
+        prefix_1 = extract_prefix(mask_1_filename)
+        prefix_2 = extract_prefix(mask_2_filename)
+        _, ext = os.path.splitext(args.in_tractogram)
+
+        fc_sft, sft = extract_false_connections(
+            sft, mask_1_filename, mask_2_filename, args.dilate_endpoints)
+
+        if len(fc_sft) > 0:
+            save_tractogram(fc_sft, os.path.join(
+                args.out_dir, '{}_{}_fc{}'.format(prefix_1, prefix_2, ext)),
+                bbox_valid_check=False)
+
+        logging.info('Recognized {} streamlines between {} and {}'.format(
+            len(fc_sft.streamlines), prefix_1, prefix_2))
+
+        fc_streamlines_list.append(fc_sft.streamlines)
+
+    nc_streamlines.extend(sft.streamlines)
+
+    final_results = {}
+    no_conn_sft = StatefulTractogram.from_sft(nc_streamlines, sft)
+    save_tractogram(no_conn_sft, os.path.join(
+        args.out_dir, 'nc{}'.format(ext)), bbox_valid_check=False)
+
+    # Total number of streamlines for each category
+    # and statistic that are not 'bundle-wise'
+    tc_streamlines_count = len(list(itertools.chain(*tc_streamlines_list)))
+    fc_streamlines_count = len(list(itertools.chain(*fc_streamlines_list)))
+
+    if args.wrong_path_as_separate:
+        wpc_streamlines_count = len(
+            list(itertools.chain(*wpc_streamlines_list)))
+    else:
+        wpc_streamlines_count = 0
+
+    nc_streamlines_count = len(nc_streamlines)
+    total_count = tc_streamlines_count + fc_streamlines_count + \
+        wpc_streamlines_count + nc_streamlines_count
+
+    assert total_count == initial_count
+
+    final_results['tractogram_filename'] = str(args.in_tractogram)
+    final_results['tractogram_overlap'] = 0.0
+    final_results['tc_streamlines'] = tc_streamlines_count
+    final_results['fc_streamlines'] = fc_streamlines_count
+    final_results['nc_streamlines'] = nc_streamlines_count
+
+    final_results['tc_bundle'] = len([x for x in tc_streamlines_list if x])
+    final_results['fc_bundle'] = len([x for x in fc_streamlines_list if x])
+
+    final_results['tc_streamlines_ratio'] = tc_streamlines_count / total_count
+    final_results['fc_streamlines_ratio'] = fc_streamlines_count / total_count
+    final_results['nc_streamlines_ratio'] = nc_streamlines_count / total_count
+
+    if args.wrong_path_as_separate:
+        final_results['wpc_streamlines'] = wpc_streamlines_count
+        final_results['wpc_streamlines_ratio'] = \
+            wpc_streamlines_count / total_count
+        final_results['wpc_bundle'] = len(
+            [x for x in wpc_streamlines_list if x])
+
+    final_results['total_streamlines'] = total_count
+    final_results["bundle_wise"] = {}
+    final_results["bundle_wise"]["true_connections"] = {}
+    final_results["bundle_wise"]["false_connections"] = {}
```

Comment:
Minor: There is a mix of " and ' for string here, a few line above you use ' for all str

Reply:
I know most people use `"`, but it's an extra keystroke (shift) compared to `'` and I'm all about saving keystrokes lol